src/rsa/signing.rs

```diff
@@ -26,6 +26,30 @@ use crate::{
 };
 use untrusted;
 
+/// Used to construct an RSA key pair from components.
+///
+/// Components are in big-endian, considered unsigned, and must not contain
```

Comment:
This documentation about how a `Positive` must be encoded should be in the `Positive` constructor, not here.